modules/nf-core/untar/main.nf

```diff
@@ -2,7 +2,7 @@ process UNTAR {
     tag "$archive"
```

Comment:
why is this main.nf touched?

Reply:
https://github.com/nf-core/modules/issues/2874